config/projects/datadog-agent.rb

```diff
@@ -115,6 +144,27 @@
   dependency 'sysstat'
 end
 
+# Ship supervisor anywhere but on Windows
+if not windows?
+  dependency 'kafka-python'
+  dependency 'python-gearman'
+  dependency 'snakebite'
+  dependency 'supervisor'
+
+  # Technically these ones should be shipped on Windows too at some point...
+  # if we ever happen to have a customer use Postgre/pg_boucer on that platform :)
+  dependency 'psycopg2'
+  dependency 'pg8000'
```

Comment:
pg8000 is a pure python dependency that we currently ship with the windows Agent, so I think we should keep shipping it on Windows (unless there's a good reason not to)